dataplane/saiserver/acl.go

```diff
@@ -250,6 +250,42 @@ func (a *acl) createAclEntryFields(req *saipb.CreateAclEntryRequest, id uint64,
 			Masks:   []byte{byte(req.GetFieldTtl().GetMaskUint())},
 		})
 	}
+	table := &saipb.AclTableAttribute{}
+	if err := a.mgr.PopulateAllAttributes(fmt.Sprint(req.GetTableId()), table); err != nil {
+		return nil, err
+	}
+	for id, field := range req.GetUserDefinedFieldGroupMin() {
+		udfGroup := &saipb.UdfGroupAttribute{}
+		if err := a.mgr.PopulateAllAttributes(fmt.Sprint(table.UserDefinedFieldGroupMin[id]), udfGroup); err != nil {
+			return nil, err
+		}
+		for _, udfID := range udfGroup.GetUdfList() {
+			udf := &saipb.UdfAttribute{}
+			if err := a.mgr.PopulateAllAttributes(fmt.Sprint(udfID), udf); err != nil {
+				return nil, err
+			}
+			hg := fwdpb.PacketHeaderGroup_PACKET_HEADER_GROUP_L2
```

Comment:
With this L2 is the default value. Is this safe?

Reply:
yes, this is intentional